### DISCUSSION TITLE
Skip pages on pageNumber when adding a cover

### DISCUSSION THREAD

**Discussion by rumeau** [spatie/laravel-pdf#149] [Q&A]
Hello. I've managed to create a PDF report. Now I want to add page numbers to internal pages of the report. I included a cover and the page numbers are starting on the cover.

How could I skip the cover in order to start the first page after the cover. Or is this possible?

**Comment by ArielMejiaDev** [comment]
@rumeau  You are able to generate two PDF files the first one with the portrait and another skipping the first page with:

```php 
Pdf::view('test')
    ->withBrowsershot(function (Browsershot $browsershot) {
        $browsershot>initialPageNumber(8);
    });
and then use a library to merge files like: https://github.com/hanneskod/libmergepdf

se iio\libmergepdf\Merger;

$portrait = Pdf::view('portrait')->save('portrait.pdf');
$content = Pdf::view('your-content')->save('content.pdf');

$merger = new Merger;
$merger->addIterator([$portrait, $content]);
$createdPdf = $merger->merge();
```

it should works
